platform/core/src/extensions/MODULE_TYPES.js

```diff
@@ -4,5 +4,5 @@ export default {
   SOP_CLASS_HANDLER: 'sopClassHandlerModule',
   TOOLBAR: 'toolbarModule',
   VIEWPORT: 'viewportModule',
-  ROUTES: 'routesModule',
+  ROUTES: 'routesTemplateModule',
```

Comment:
plural routes, singular template?
This may need to be: `routeTemplatesModule`

Reply:
ok I will change it